app/styles/main.scss

```
@@ -1 +1,122 @@
 // import "base";
+
+/* colors */
+
+$primary-color: #2196f3;
+$accent-color: #e84855;
+$footer-color: #403f4c;
+
+.panel {
+  border: 2px solid $primary-color;
+  border-radius: 5px;
+}
+
+.panel > .panel-heading {
+  background-color: $primary-color;
+  color: white;
+  margin: 0px;
+  padding: 5px;
+}
+
+.text-primary {
+  color: $primary-color;
+}
+
+.footer {
+  background-color: $footer-color;
+  color: white;
+  padding: 30px 0px;
+}
+
+.focus {
+  background-color: $accent-color;
+  padding: 4px;
+  border-radius: 3px;
+}
+
+nav {
+  background-color: $primary-color;
+  color: white;
+  padding: 0px 30px;
+  a:hover, a:link, a:visited, a:active {
+    color: white;
+    text-decoration: none;
+  }
+  li {
+    margin: 10px;
+  }
+  .nav-right {
+    position: absolute;
+    right: 0px;
+  }
+}
+
+/* layout */
+
+.panel > .panel-body {
+  padding: 5%;
+}
+
+.visible-lg {
+  visibility: hidden;
```

Better to keep it visible by default and hide in smaller screens.

---

But I thought we were going mobile-first?